web/help/rlt-notes-unit1.md

```diff
@@ -0,0 +1,14 @@
+If you do not leave a space before and after the & the popup does not show
+If the student makes change and clicks an example they get a warning. It might be beneficial to mention something about this and/or saving.
+
+Should variable really be constant since you cannot re-assign it a new value?
```

Comment:
Variables are commonly used in mathematics, and can never be re-assigned new values.  We should call them variables.

Reply:
Agreed. Again, me thinking in programming terms like const.